oxide.nix

```diff
@@ -14,17 +14,38 @@ stdenv.mkDerivation {
   ];
 
   preBuild = ''
-    source ${remarkable-toolchain}/environment-setup-cortexa9hf-neon-oe-linux-gnueabi
+    source ${remarkable-toolchain}/environment-setup-cortexa9hf-neon-remarkable-linux-gnueabi
   '';
 
   enableParallelBuilding = true;
   makeFlags = [ "release" ];
 
   installPhase = ''
-    cp -r release/. $out
+    install -D -m 755 -t $out/opt/bin release/opt/bin/erode
```

Comment:
This looks like it's going to be painful to maintain. Is there a reason you made it explicit instead of just using the entire contents of the release folder?

Reply:
@siraben poke?